src/engine/module.c

```diff
@@ -327,7 +327,7 @@ dss_module_cleanup_all(void)
 		if (rc != 0) {
 			D_ERROR("failed to clean up module %s: "DF_RC"\n",
 				m->sm_name, DP_RC(rc));
-			break;
+			/** continue clean-ups regardless ... */
```

Comment:
Should we use a continue here so we don't display the `"Module %s: cleaned up\n"` message?
```suggestion
			continue;
```

Reply:
Well, it will just print an extra message that the module cleanup has happened. Pretty harmless.

Reply:
Ok